Troubleshooting.md

```diff
@@ -111,7 +111,7 @@ Then run `pip install future`.
 
 ### No Java
 
-If running `which java` on the terminal does not return any output, you do not have java installed. You can install it by running `sudo apt install openjdk-7-jre-headless`. Note that this command might vary for your local machine.
+If running `which java` on the terminal does not return any output, you do not have java installed. You can install it by running `sudo apt install openjdk-11-jre`. Note that this command might vary for your local machine.
```

Comment:
Same as other comment.

Reply:
Ah, just saw the LGTM. Thanks, I'll go ahead and merge!